### PR TITLE
add protocol council charter

### PR DESCRIPTION
**Pull request by abbey-titcomb**
add charter for protocol council

### CODE REVIEW

File: protocol-council-charter.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+authorized, ex ante, to act on behalf of the community in exigent
+circumstances; (iv) the influence and impact of malicious or negligent
+actors are mitigated through cautious EigenGov design and
+implementation; and ultimately, (vi) the Council in its current form
+should be made obsolete, reflecting our commitment to responsible
+decentralization and protocolized governance over time.
+
+## **Exhibit A**
+
+### **EigenLayer Core Contracts**
```
Comment (non-fungible-nelson) [inline protocol-council-charter.md:200]: Should this reference the source of truth file at https://github.com/eigenfoundation/ELIPs/blob/main/contracts.md? That way we can track just in that one file for consistency.

Might require updating contracts.md

File: protocol-council-charter.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+The Council may invite to its meetings non-Council members, including
+any persons the Council deems appropriate in order to carry out its
+responsibilities.  
+
+#### **6. Conflicts of Interest**
+
+Prior to exercising voting and/or Multisig authority, the Council must
+review all relevant facts and circumstances among the members to
+determine whether a material conflict of interest may arise due to a
+member's [family, social, financial, or professional]{.mark} interests
```
Comment (non-fungible-nelson) [inline protocol-council-charter.md:139]: these {.mark} are not displaying correctly

File: protocol-council-charter.md (hunk 1 above)
Comment (abbey-titcomb) [reply protocol-council-charter.md:200]: Oh yes, good catch. I will update contracts.md with this

Comment (abbey-titcomb) [reply protocol-council-charter.md:200]: updated!

File: protocol-council-charter.md (hunk 2 above)
Comment (abbey-titcomb) [reply protocol-council-charter.md]: fixed

**Review by non-fungible-nelson — APPROVED** [review]
LGTM 🚢

**Pull request merged**